src/camvidlog2/queries.py

```
@@ -0,0 +1,75 @@
+    # combine the embeddings from strings and frames, in the original order!
+
+    interleafing = [
+        (
+            nda_strings[i, :] if isinstance(e, StringEmbedding) else None,
+            nda_frames[i] if isinstance(e, FrameEmbedding) else None,
+        )
+        for i, e in enumerate(embedding_group.items)
+    ]
+
+    nda_combined = pd.DataFrame(
+        np.stack([a if a is not None else b for a, b in interleafing])
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)